include/seqan3/alignment/pairwise/policy/affine_gap_banded_policy.hpp

```diff
@@ -72,25 +72,53 @@ class affine_gap_banded_policy : public affine_gap_policy<affine_gap_banded_poli
     {
         using std::get;
         // Unpack the cached variables.
-        auto & [current_entry, next_entry] = get<0>(current_cell);
-        auto & [main_score, hz_score] = current_entry;
-        auto const & prev_hz_score = get<1>(next_entry);
+        auto & [score_entry, coordinate, trace_value] = current_cell;
+        auto & [current_entry, next_entry] = score_entry;
+        auto & [main_score, hz_score, hz_trace] = current_entry;
+        auto const & [_ignore, prev_hz_score, prev_hz_trace]= next_entry;
         auto & [prev_cell, gap_open, gap_extend, opt] = cache;
-        auto & [tmp, vt_score] = prev_cell;
+        auto & [tmp, vt_score, vt_trace] = prev_cell;
```

Comment:
Find a better name for tmp.

Reply:
But it is a value that is not propagated out of the function and is used to temporarily store an intermediate result. So that is what `tmp` suggests